src/bp-forums/activity.php

```diff
@@ -186,15 +196,14 @@ private function fully_loaded() {
 		 * @uses bp_activity_set_action()
 		 */
 		public function register_activity_actions() {
-
 			// Sitewide activity stream items
 			bp_activity_set_action( $this->component, $this->topic_create, esc_html__( 'New forum discussion', 'buddyboss' ), array( $this, 'topic_activity_action_callback' ) );
 			bp_activity_set_action( $this->component, $this->reply_create, esc_html__( 'New forum reply', 'buddyboss' ), array( $this, 'reply_activity_action_callback' ) );
 
 			if ( bp_is_active( 'groups' ) ) {
-				// Group activity stream items
-				bp_activity_set_action( buddypress()->groups->id, $this->topic_create, esc_html__( 'New forum discussion', 'buddyboss' ), array( $this, 'group_forum_topic_activity_action_callback' ) );
-				bp_activity_set_action( buddypress()->groups->id, $this->reply_create, esc_html__( 'New forum reply', 'buddyboss' ), array( $this, 'group_forum_reply_activity_action_callback' ) );
+				// Group activity stream items.
+				bp_activity_set_action( 'groups', $this->topic_create, esc_html__( 'New forum discussion', 'buddyboss' ), array( $this, 'topic_activity_action_callback' ) );
+				bp_activity_set_action( 'groups', $this->reply_create, esc_html__( 'New forum reply', 'buddyboss' ), array( $this, 'reply_activity_action_callback' ) );
```

Comment:
@asaquzzamanbb What's the reason to remove the `buddypress()->groups->id` and add the `groups` static? It will break if some customers used different id for the groups?

Reply:
Remove groups here
https://github.com/buddyboss/buddyboss-platform/pull/2290/commits/fd11688f84b50e6365aba867c3fc7743fe179507